staging/src/k8s.io/apimachinery/pkg/util/runtime/runtime.go

```diff
@@ -109,11 +129,6 @@ func HandleError(err error) {
 	}
 }
 
-// logError prints an error with the call stack of the location it was reported
-func logError(err error) {
```

Comment:
Leave this as an option.  Just in case someone wants it.

Reply:
Yes

Reply:
I would have to make it public for someone to use it.  Is it really worth it since anyone that wants this can write the one-liner (or, more likely, write a more specific logger for their own use case)?